ondewo/nlu/intent.proto

```diff
@@ -968,4 +971,30 @@ message DeleteTrainingPhraseRequest {
     // Required. The name of the training phrase.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>/trainingPhrases/<Training Phrase ID>`
     string name = 1;
+}
+
+// The request message for TrainingPhraseRequest
+message ListTrainingPhraseRequest {
```

Comment:
Careful.
These changes do not belong to this PR.

Make sure to merge the changes from the other PR when resolving conflicts (if the situation arises)

Reply:
Indeed, I'll take care of that.